ProjFS.Mac/PrjFSKext/ProviderMessaging.cpp

```diff
@@ -180,7 +180,7 @@ bool ProviderMessaging_TrySendRequestAndWaitForResponse(
         if (0 != sendError)
         {
             // TODO: appropriately handle unresponsive providers
-            *kauthResult = KAUTH_RESULT_DEFER;
+            *kauthResult = KAUTH_RESULT_DENY;
```

Comment:
This change seems unrelated to the PR, but I think makes sense.  Can you confirm?
If we can't send a message fail the request.

Reply:
It's not really unrelated. Prior to this PR, we've been allowing operations to go ahead under all circumstances. In this PR, we want to deny operations if allowing them to go ahead would mean that the provider misses messages. So the goal is to deny access regardless of the *reason* for the missed message - whether this is an offline provider or some other message delivery problem.

Reply:
"or some other message delivery problem" this bit could have been existing.

I think it's a fine change and makes sense in this PR.

Reply:
I've clarified with a comment, but yeah, I think we should leave this in.